### DISCUSSION TITLE
isolated installation virtualenv-style?

### DISCUSSION THREAD

**Discussion by kov** [Homebrew/discussions#1252] [Getting started]
Hello. I got an M1 and I would like to be able to have a x86-64 install of homebrew alongside my native one. Is there a recommended way of having isolated installs? I first thought of a chroot, but Mac OS X seems to make creating a working chroot very difficult with how system libraries are handled since a few releases ago, I wasn't able to figure out how to do it. I wonder if brew itself has any facilities that would allow me to have two separate environments, the default one and one I'd have to manually enter.

I'm also open to other suggestions of how to achieve this =)

**Comment by gromgit** [comment]
I don't own an M1 (yet), but since no one corrected me the last time I posted it, try this:
```
arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
That should force the Homebrew installer to run under Rosetta auspices, and therefore install Intel binaries under `/usr/local`, while your original M1 Homebrew installation still resides under `/opt/homebrew`.

As for "manual entry", simply keep your `PATH` straight: `/usr/local/bin` for Intel stuff, `/opt/homebrew/bin` for M1 stuff.

**Reply by kov** [reply]
Nice, thank you! Will give it a try and report back if it works =)
